cinema/views.py

```diff
@@ -20,25 +22,47 @@
     OrderListSerializer,
 )
 
+from user.permissions import IsAdminOrIfAuthenticatedReadOnly
```

Comment:
Ensure that the `IsAdminOrIfAuthenticatedReadOnly` permission class is correctly implemented in `user.permissions`. This custom permission should allow read-only access to authenticated users and full access to admin users.